### PR TITLE
Add Extreme platform with 6 templates

### PR DESCRIPTION
**Pull request by PavloSkliarenko**
New Extrem platform with the templates for 6 commands:

`show ipconfig`
`show ports description`
`show ports information`
`show ports information detail`
`show sharing`
`show vlan description`

### CODE REVIEW

File: ntc_templates/templates/extreme_show_ports_description.textfsm [hunk 1]
```diff
@@ -0,0 +1,18 @@
+Value PORT (\d+)
```
Comment (jvanderaa) [inline ntc_templates/templates/extreme_show_ports_description.textfsm:1]:
```suggestion
Value INTERFACE (\d+)
```

File: ntc_templates/templates/extreme_show_ports_information.textfsm [hunk 2]
```diff
@@ -0,0 +1,41 @@
+Value PORT (\d+)
```
Comment (jvanderaa) [inline ntc_templates/templates/extreme_show_ports_information.textfsm:1]:
```suggestion
Value INTERFACE (\d+)
```

File: ntc_templates/templates/extreme_show_ports_information_detail.textfsm [hunk 3]
```diff
@@ -0,0 +1,26 @@
+Value PORT (\d+)
```
Comment (jvanderaa) [inline ntc_templates/templates/extreme_show_ports_information_detail.textfsm:1]:
```suggestion
Value INTERFACE (\d+)
```

File: ntc_templates/templates/extreme_show_vlan_description.textfsm [hunk 4]
```diff
@@ -0,0 +1,17 @@
+Value NAME (\S+)
+Value ID (\d+)
```
Comment (jvanderaa) [inline ntc_templates/templates/extreme_show_vlan_description.textfsm:2]:
```suggestion
Value VLAN_NAME (\S+)
Value VLAN_ID (\d+)
```

**Review by jvanderaa — CHANGES_REQUESTED** [review]
Updates for the data model of 4.0

**Review by jvanderaa — APPROVED** [review]

**Comment by mayafox** [comment]
This does not correctly identify the OS of the Extreme device in use.
Is this Extreme TierraOS? VOSS? XOS? NOS? SLXOS? Wing?

**Comment by PavloSkliarenko** [comment]
@mayafox This is ExtremeXOS (EXOS)

**Comment by mayafox** [comment]
> @mayafox This is ExtremeXOS (EXOS)

If that's the case, the file names of the templates and test files should  reflect that 
The file name format  is <mfg>_<platform>_<underscored command> 
for example, the show ports description command  it would be  extreme_exos_show_ports_description.textfsm

This allows the NTC code templates and textfsm  to differentiate between the platforms from the same mfg that have different output formats for the same command.

**Comment by PavloSkliarenko** [comment]
@mayafox Fix platform name  in this [PR](https://github.com/networktocode/ntc-templates/pull/1576)

**Pull request merged**